controlpanel/settings/common.py

```diff
@@ -456,16 +456,25 @@
 REDIS_HOST = os.environ.get('REDIS_HOST', 'localhost')
 REDIS_PASSWORD = os.environ.get('REDIS_PASSWORD')
 REDIS_PORT = os.environ.get('REDIS_PORT', '6379')
+REDIS_SCHEME = os.environ.get("REDIS_SCHEME", "redis")
+if REDIS_SCHEME not in ["redis", "rediss"]:
+    raise ValueError(f"Invalid value for 'REDIS_SCHEME' environment variable. Must be 'redis' or 'rediss' (to use SSL/TLS). It was '{REDIS_SCHEME}' which is invalid.")
+
+REDIS_URI = f"{REDIS_SCHEME}://{REDIS_HOST}:{REDIS_PORT}/1"
```

Comment:
Why the digit `1` at the end of this URI..? (It didn't used to have one... no pun intended).